modules/pico_tcp.c

```diff
@@ -54,12 +54,16 @@
 
 
 #ifdef PICO_SUPPORT_TCP
-#define tcp_dbg_nagle(...) do {} while(0)
-#define tcp_dbg_options(...) do {} while(0)
-
 
-#define tcp_dbg(...) do {} while(0)
-/* #define tcp_dbg dbg */
+#ifdef DEBUG_TCP
+#define tcp_dbg_nagle        dbg
+#define tcp_dbg_options      dbg
+#define tcp_dbg              dbg
```

Comment:
I added the separate tap device macros (f912de3d29c20519caa825ffc560f374b002fc62). For the MM, it indeed looks like it's not the same as with tap or tcp. So I guess separate debug macros would be overkill here.